its/plugin/src/test/java/com/sonar/it/web/HtmlTestSuite.java

```diff
@@ -66,39 +65,34 @@ public static SonarScanner createSonarScanner() {
     return SonarScanner.create();
   }
 
-  @CheckForNull
-  static Measure getMeasure(Orchestrator orchestrator, String componentKey, String metricKey) {
-    WsMeasures.ComponentWsResponse response = newWsClient(orchestrator).measures().component(new ComponentWsRequest()
-      .setComponentKey(componentKey)
-      .setMetricKeys(Collections.singletonList(metricKey)));
-    List<Measure> measures = response.getComponent().getMeasuresList();
+  static Measures.Measure getMeasure(Orchestrator orchestrator, String componentKey, String metricKey) {
+    Measures.ComponentWsResponse response = newWsClient(orchestrator).measures().component(new ComponentRequest()
+      .setComponent(componentKey)
+      .setMetricKeys(singletonList(metricKey)));
+    List<Measures.Measure> measures = response.getComponent().getMeasuresList();
     return measures.size() == 1 ? measures.get(0) : null;
   }
 
   @CheckForNull
   static Integer getMeasureAsInt(Orchestrator orchestrator, String componentKey, String metricKey) {
-    Measure measure = getMeasure(orchestrator, componentKey, metricKey);
+    Measures.Measure measure = getMeasure(orchestrator, componentKey, metricKey);
     return (measure == null) ? null : Integer.parseInt(measure.getValue());
   }
 
   @CheckForNull
   static Double getMeasureAsDouble(Orchestrator orchestrator, String componentKey, String metricKey) {
-    Measure measure = getMeasure(orchestrator, componentKey, metricKey);
+    Measures.Measure measure = getMeasure(orchestrator, componentKey, metricKey);
     return (measure == null) ? null : Double.parseDouble(measure.getValue());
   }
 
+
   @CheckForNull
-  static Component searchComponent(Orchestrator orchestrator, String projectKey, String componentKey) {
-    List<Component> components = newWsClient(orchestrator).components().tree(
-      new TreeWsRequest()
-        .setBaseComponentKey(projectKey)
-        .setQuery(componentKey))
+  static Components.Component searchComponent(Orchestrator orchestrator, String componentKey) {
+    List<Components.Component> components = newWsClient(orchestrator).components()
+      .search(new SearchRequest().setQ(componentKey).setQualifiers(singletonList("FIL")))
```

Comment:
please add a comment for what `"FIL"` means

Reply:
not a big thing, but `"FIL"` is a little bit cryptic to me. Do we have documentation for it somewhere?

Reply:
I added the documentation, it was merged in another PR https://github.com/SonarSource/sonar-html/blob/master/its/plugin/src/test/java/com/sonar/it/web/HtmlTestSuite.java#L91 

this is document in the webservice documentation, but I agree it is bit cryptic